pkg/useragent/analyzer.go

```diff
@@ -60,21 +60,23 @@ func (a *analyzer) start() {
 			// set up writer output
 			var output update
 
-			if len(datum.OrigIps) > 10 {
-				datum.OrigIps = datum.OrigIps[:10]
+			origIPs := datum.OrigIps.Items()
+			if len(origIPs) > 10 {
+				origIPs = origIPs[:10]
 			}
 
-			if len(datum.Requests) > 10 {
-				datum.Requests = datum.Requests[:10]
+			requests := datum.Requests.Items()
```

Comment:
Note that `.Items()` must be called in order to convert the map into an array. 
https://github.com/activecm/rita/commit/c4b5b3fa83b503af6da624a9727cbf1bfbaaa45a